app/src/pages/homes.tsx

```diff
@@ -0,0 +1,69 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import React from "react";
+import Image from "next/image";
+import { trpc } from "utils/trpc";
+import Navbar from "@components/navbar";
+import { useState } from 'react';
+
+
+const HomesPage: NextPage = () => {
+    const [error, setError] = useState<string | null>(null);
+
+    
+    const homes = trpc.useQuery(["home.getHomes"]);
+
+
+    return (
+        <>
+            <Head>
+                <title>RBH Homes Page</title>
+                <meta
+                    name="description"
+                    content="Homes page of Roommate budget helper"
+                />
+            </Head>
+            <div className="body flex flex-col text-center">
+                <Navbar />
+                <div className="text-xl p-5">
+                    <div className="form-area flex flex-col justify-between items-center">
+                        <div className="p-5">
+                            Welcome to Roomate Budget Helper <a className="text-5xl">👋</a>
+                        </div>
+                        <div className="w-1/5 m-auto h-full p-3">
+                            <Image
+                                src="/images/logo.png"
+                                alt="Home"
+                                width="100px"
+                                height="100px"
+                            />
+                        </div>
+                        {homes.data && homes.data.length > 0 ? <>
+                        <div>You belong to {homes.data.length} homes</div>
+                        <br></br>
+                        
+                        <div id="homeDisplay">
```

Comment:
Not for sure why the id is specified, couldn't find it anywhere else in the project? Maybe future use?

Reply:
yeah I think that is a leftover from when Curtiss was trying to do querySelectors, it doesn't really have any use right now but may be useful for doing end-to-end testing later on but it may be good to remove if we want to be more consistent